### PR TITLE
feat!: updated twitter (X) icons

### PR DESCRIPTION
**Pull request by eshet-hayil**
- [x] **BREAKING CHANGE?** 

## Checklist

* [x] :ok_hand: SVG updates are Garden Designer approved (add the
  designer as a reviewer)
* [ ] :globe_with_meridians: SVG demo is up-to-date (`npm start`)
* [ ] :black_medium_small_square: Renders as expected in "dark" mode
* [ ] :white_large_square: Renders as expected @ 2x scale

### CODE REVIEW

File: .DS_Store
Comment (steue) [inline .DS_Store:1]: These `src/.DS_Store` files snuck in here. these are OS specific files we dont include in our repos.

File: src/.DS_Store
Comment (steue) [inline src/.DS_Store:1]: same. remove.

File: src/12/.DS_Store
Comment (steue) [inline src/12/.DS_Store:1]: ^

**Review by steue — CHANGES_REQUESTED** [review]
the icons look good to me! need to remove the `.DS_Store` files, but other than that, looks ready to go! good work

**Review by geotrev — COMMENTED** [review]
Other than the `.DS_Store` files, looks good. Will approve when those are gone. :)

**Review by steue — APPROVED** [review]
noice

**Review by jzempel — APPROVED** [review]
Great quality on the SVG source. Thanks.

**Pull request merged**